test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -1071,17 +1071,17 @@ int run_test_with_multi_import_diff_ctx(
     }
     else
     {
-        clVk2CLExternalSemaphore = new clExternalSemaphore(
+        clVk2CLExternalSemaphore = new clExternalImportableSemaphore(
             vkVk2CLSemaphore, context, vkExternalSemaphoreHandleType, deviceId);
-        clCl2VkExternalSemaphore = new clExternalSemaphore(
+        clCl2VkExternalSemaphore = new clExternalExportableSemaphore(
             vkCl2VkSemaphore, context, vkExternalSemaphoreHandleType, deviceId);
```

Comment:
Are we checking somewhere for supported exportable handle types before creating exportable semaphore?
If exportable handle types is empty, may be we should consider falling back to importable semaphore for this semaphore too.